source/AI.cpp

```diff
@@ -4065,3 +4066,16 @@ void AI::UpdateOrders(const Ship &ship)
 		order.targetSystem = ship.GetSystem();
 	}
 }
+
+
+void AI::SetInvisibleFenceRadius(double radius)
+{
+	this->invisibleFenceRadius = radius;
+}
+
+
+
+double AI::GetInvisibleFenceRadius() const
+{
+	return this->invisibleFenceRadius;
```

Comment:
```suggestion
	return invisibleFenceRadius;
```

I mean this function will be uneccessary if you look at my last comment, but for the future, class members do not need to be accessed with this->